test/e2e/atlas/operator_helper_test.go

```diff
@@ -66,36 +66,26 @@ func newOperatorHelper(t *testing.T) (*operatorHelper, error) {
 	}, nil
 }
 
-func createCluster(name string) error {
+func createK8SCluster(name string) error {
```

Comment:
FYI @mongodb/atlas_kubernetes_team I renamed this as it confused me with deleting an Atlas cluster